src/graph_impl/stable_graph/mod.rs

```diff
@@ -635,10 +635,7 @@ impl<N, E, Ty, Ix> StableGraph<N, E, Ty, Ix>
         }
     }
 
-    /// Return an iterator of all edges of `a`.
-    ///
-    /// - `Directed`: Outgoing edges from `a`.
-    /// - `Undirected`: All edges connected to `a`.
+    /// Return an iterator of all outgoing edges of `a`.
```

Comment:
I'm curious, in what way was the old doc misleading? It's important to note how the directedness (or not) of the graph's edges influences the behaviour here.

Reply:
I get it now. So this doc means that for different kinds of graphs, `edges` has different behavior. I misunderstood that